__tests__/utils/resize-observer.tsx

```diff
@@ -0,0 +1,14 @@
+export class ResizeObserver {
```

Comment:
This is to add on the tesing file everytime we have a slider on our component or page that's it?
Can you add this info in the testing tips doc?